clang/test/DebugInfo/KeyInstructions/init-member-memcopyable-2.cpp

```diff
@@ -0,0 +1,59 @@
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -gkey-instructions %s -gno-column-info -debug-info-kind=line-tables-only -emit-llvm -o - \
+// RUN: | FileCheck %s
+
+// g::h and i can be memcpy'd, check the assignment gets Key Instructions metadata.
+
+struct e {
+  e(e &);
+  e& operator=(const e&);
+};
+
+struct g {
+  e f;
+  int h;
+  int i;
+};
+
+// Copy assignment operator.
+// CHECK: define{{.*}}ptr @_ZN1gaSERKS_
```

Comment:
This'll trip on Windows machines IMO, there's a `%clang_cc1` macro that ensures you're targetting an Itanium ABI I think?

Reply:
Isn't that already covered by `-triple x86_64-linux-gnu`? Happy to change it to `%itanium_abi_triple` if you think that's better?